tests/network/primary_pod_network.go

```diff
@@ -105,21 +105,6 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 					Eventually(vmiIPs, 2*time.Minute, 5*time.Second).Should(Equal(podIPs), "should contain VMI Status IP as Pod status IPs")
 				})
 			})
-			When("no Guest Agent exists", func() {
```

Comment:
The commit message states:
> The original test use Alpine VM that has no QEMU-GA installed, the target test
use Cirros VM which doesn't have QEMU-GA installed as well.

How about:
> Both the test removed by this commit and `[test_id:1540]` use OS images that do not contain QEMU-GA.

Reply:
Done

Reply:
nice to have: It would have been easier to track the alternative test using a permanent link (from the last tagged release).